### ISSUE TITLE
the argument of  num_epochs seem not work

### ISSUE THREAD

**Issue by PamelaDDD** [jialinwu17/MAVEX#3]
It seems that the epoch id should be changed to be greater than or equal to args.num_epochs, otherwise once the epoch_id increases to greater than num_epoch in the inner loop, it will not jump out of the outer loop。

**Comment by PamelaDDD** [comment]
![image](https://user-images.githubusercontent.com/33123364/159828367-d6de793a-f8c4-4f20-bf6c-e619ac958013.png)
![image](https://user-images.githubusercontent.com/33123364/159828463-f3e6eabb-ac94-4358-849e-16f6a5f6a8ea.png)

**Comment by jialinwu17** [comment]
Yes, you are right. I will update the training scripts. Fortunately, we only have one task during finetuning.

**Comment by PamelaDDD** [comment]
Thanks for your reply. Although there is only one finetune task，it dosen't stop when epoch_id equal with args.num_epochs when the program running with the num_epochs argument, and the saved model doesn't seem to be the one at epoch 75,




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "jialinwu17/MAVEX"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年3月24日(星期四) 上午10:28
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [jialinwu17/MAVEX] the argument of  num_epochs seem not work (Issue #3)





 
Yes, you are right. I will update the training scripts. Fortunately, we only have one task during finetuning.

**Comment by jialinwu17** [comment]
Let me take a deeper dive into the code, thanks for bringing that up

**Comment by jialinwu17** [comment]
I see the problem, will update this
